src/puppetlabs/ezbake/core.clj

```diff
@@ -568,6 +569,8 @@ Additional uberjar dependencies:
                                                       "puppetlabs.trapperkeeper.main")
      :java-args                          (get-local-ezbake-var lein-project :java-args
                                                       "-Xmx192m")
+     :tk-args                            (get-local-ezbake-var lein-project :tk-args
+                                                       "" )
```

Comment:
@mwaggett updated. I had to set it to an empty string otherwise it gets mad
```
Error encountered performing task 'ezbake' with profile(s): 'ezbake'
clojure.lang.ArityException: Wrong number of args (2) passed to: core/get-local-ezbake-var
	at clojure.lang.AFn.throwArity(AFn.java:429)
	at clojure.lang.AFn.invoke(AFn.java:36)
	at puppetlabs.ezbake.core$make_template_map.invokeStatic(core.clj:572)
```

Reply:
I'd like to know what that value is used for. Is it a default?

Reply:
Yes, it's used when creating the ezbake.rb template. I'm waiting on the ticket creator to get what the default value should be.

Reply:
Empty string is a good default. Most common use of this would be to get the `--debug` flag passed into TK daemon(s) when needed.